Scripts/update_ncbi_codon_table.py

```diff
@@ -16,14 +19,20 @@
 import re
 
 INDENT = len("register_ncbi_table(")
+INDENT2 = INDENT + len("table={")
 
 
-def line_wrap(text, indent=0, max_len=78):
+def line_wrap(text, indent=0, max_len=78, string=False):
+    """Return a wrapped line if lenght is larger max_len."""
```

Comment:
Typo (lenght)

Reply:
Also might be worth expanding the docstring to explain the use of the new string argument - I can see you use this to split string arguments over multiple lines, but it would only work if the strings are single quote delimited.

(The implementation is fine given it is specific to this helper script, but making this more explicit seems safer for future users of this code.)